src/meta-srv/src/procedure/region_migration.rs

```diff
@@ -127,6 +127,8 @@ pub struct VolatileContext {
     leader_region_lease_deadline: Option<Instant>,
     /// The last_entry_id of leader region.
     leader_region_last_entry_id: Option<u64>,
+    /// The last_entry_id of leader metadata region (Only used for metrics).
```

Comment:
```suggestion
    /// The last_entry_id of leader metadata region (Only used for metric engine).
```

Reply:
resolved in [cb27076](https://github.com/GreptimeTeam/greptimedb/pull/5842/commits/cb27076f79c8421627c23b041cb1dd2e79a9905e)